index.d.ts

```diff
@@ -17,6 +17,10 @@ isAbsoluteUrl('foo/bar');
 //=> false
 ```
 */
-declare function isAbsoluteUrl(url: string): boolean;
+interface Options {
```

Comment:
This needs to be inside a namespace: https://github.com/sindresorhus/typescript-definition-style-guide